Tools/autotest/arducopter.py

```
@@ -924,7 +924,33 @@ def fly_mission(mavproxy, mav, height_accuracy=-1.0, target_altitude=None):
     wait_mode(mav, 'LOITER')
     return ret
 
-
+def fly_motor_fail(mavproxy, mav, fail_servo=0, fail_mul=1.0, holdtime=30):
+    """Fly a mission from a file."""
+    global homeloc
+    global num_wp
+    if (fail_servo < 0) or (fail_servo > 6):
```

Raise an exception on invalid input instead of silently fixing it.